src/controllers/check.your.answers.controller.ts

```diff
@@ -33,8 +35,11 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
       templateName: config.CHECK_YOUR_ANSWERS_PAGE,
       hasTrusts,
       appData,
+      changeLinkUrl,
+      pageTitle,
       pageParams: {
-        isTrustFeatureEnabled: isActiveFeature(config.FEATURE_FLAG_ENABLE_TRUSTS_WEB)
+        isTrustFeatureEnabled: isActiveFeature(config.FEATURE_FLAG_ENABLE_TRUSTS_WEB),
+        isRegister: 'true'
```

Comment:
isRegistration (I'm not sure that's much clearer but I was reading this as being related to the company register)